applications/tari_dan_wallet_cli/src/command/nfts.rs

```diff
@@ -56,12 +64,28 @@ pub struct MintAccountNftArgs {
     pub create_account_nft_fee: Option<u32>,
 }
 
+#[derive(Debug, Args, Clone)]
+pub struct GetAccountNftArgs {
+    #[clap(long, short = 'i', alias = "nft_id")]
+    pub nft_id: String,
+}
+
+#[derive(Debug, Args, Clone)]
+pub struct ListAccountNftArgs {
+    #[clap(long, short = 'l', alias = "limit")]
```

Comment:
nit: alias to the same name as the field is redundant
```suggestion
    #[clap(long, short = 'l')]
```

Reply:
Nice !